customer-commons/src/main/java/no/unit/nva/customer/model/CustomerList.java

```diff
@@ -18,34 +22,25 @@ public CustomerList() {
 
     }
 
-    public CustomerList(List<CustomerDto> customers) {
+    public CustomerList(URI id, List<CustomerDtoWithoutContext> customers) {
```

Comment:
I think it would be simpler if CustomerList accepted CustomerDtos.
```
    public CustomerList(List<CustomerDto> customers) {
        this.id = NAMESPACE;
        this.customers = extractCustomers(customers);
        this.context = CustomerMapper.CONTEXT;
    }
    private List<CustomerDtoWithoutContext> extractCustomers(List<CustomerDto> customers) {
        return Optional.ofNullable(customers)
            .stream()
            .filter(Objects::nonNull)
            .flatMap(Collection::stream)
            .filter(Objects::nonNull)
            .map(CustomerDto::withoutContext)
            .collect(Collectors.toList());
    }
```

Reply:
There is a problem with a test expecting a non empty list containing a null object.